apps/browser-extension/src/background/background.ts

```diff
@@ -56,7 +56,13 @@ async function handleContextMenuClick(info: chrome.contextMenus.OnClickData) {
     });
   } else if (menuItemId === ADD_LINK_TO_HOARDER_ID) {
     let newBookmark: ZNewBookmarkRequest | null = null;
-    if (info.selectionText) {
+    if (info.linkUrl) {
+      newBookmark = {
+        type: BookmarkTypes.LINK,
+        url: info.linkUrl,
+        title: info.selectionText ?? undefined,
+      };
+    } else if (info.selectionText) {

```

Comment:
hmmm, the ordering of the `ifs` makes me think that having text selection taking higher priority was some intentional decision. I'll need to remember why it was done that way :)

Reply:
Haha. Take your time. :)

Reply:
any update on this?